README.md

```diff
@@ -128,6 +128,14 @@ jobs:
         port: 8080
 ```
 
+## <a id="multi-arch-builds">Multi arch builds</a>
```

Comment:
sorry i wasn't clear. Place the anchor above the header, don't make the header an anchor. We don't need it to be clickable.

```html
<a id="multi-arch-builds"></a>
## Multi arch builds
```